### PR TITLE
Add explore page

### LINKED ISSUE #19: Create explore page

### PR DESCRIPTION
**Pull request by michael-caira**
Closes #19 

Added explore page. Won't show unless a user has images that have a tag so here's what it looks like with some dummy data:
Screenshot 2022-11-14 183914

@notendurable, modals that pop on click can be found in the Collection.tsx component so you can edit them there, feel free to move it around if necessary though.

### CODE REVIEW

File: client/src/pages/Explore.tsx [hunk 1]
```diff
@@ -1,5 +1,103 @@
...
+        }
+    };
+
+    useEffect(() => {
+        gatherTags();
+    }, []);
+
     return (
-        <div>Explore</div>
+        <Flex align={['center', 'center', 'normal', 'normal']} direction='column' p={['1rem', '2rem', '4rem', '4rem']}>
```
Comment (patel1643) [inline client/src/pages/Explore.tsx:61]: appreciate the responsive configuration for all components :)

**Review by patel1643 — APPROVED** [review]
LGTM

**Comment by Mwpereira** [comment]
> Closes #19
> 
> Added explore page. Won't show unless a user has images that have a tag so here's what it looks like with some dummy data: Screenshot 2022-11-14 183914
> 
> @notendurable, modals that pop on click can be found in the Collection.tsx component so you can edit them there, feel free to move it around if necessary though.

Wow 🤩

File: client/src/pages/Explore.tsx (hunk 1 above)
Comment (michael-caira) [reply client/src/pages/Explore.tsx:61]: learned/copy pasted from the best lol

**Pull request merged**